examples/remoteBuild.md

```diff
@@ -26,7 +26,7 @@ someone who wants to send them all at the same time):
 
 <!-- @remoteBuild @testAgainstLatestRelease -->
 ```
-target="https://github.com/kubernetes-sigs/kustomize/examples/multibases?ref=v1.0.6"
```

Comment:
kustomize actually clones the repo, and doesn't use this URL in the traditional sense